packages/ibm-products/src/components/ConditionBuilder/ConditionBuilderContext/DataConfigs.js

```diff
@@ -106,15 +112,25 @@ export const valueRenderers = {
     return config.unit && val ? `${val} ${config.unit}` : val;
   },
   option: (value) => {
-    return Array.isArray(value) ? value.join(', ') : value;
+    if (value && typeof value == 'string') {
+      return value;
+    } else if (value) {
+      const selectedValues = value && Array.isArray(value) ? value : [value];
```

Comment:
`value &&` is unnecessary since you're already checking `value` above

Reply:
modified and did some refractor